lib/core/utils.js

```diff
@@ -499,7 +489,7 @@ RESUtils.thing = function(element) {
 RESUtils.thing.prototype = {
 	thingSelector: '.thing, .search-result-link',
 	entrySelector: '.entry, .search-result-link > :not(.thumbnail)',
-	containerSelector: '.sitetable, .search-result-listing',
+	containerSelector: '.sitetable, .search-result-listing:last',
```

Comment:
may need to do `:last-of-type` to support https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/2671/files#r50610107